cmd/virt-launcher/virt-launcher.go

```diff
@@ -238,6 +239,12 @@ func main() {
 
 	log.InitializeLogging("virt-launcher")
 
+	// Block until all requested hookSidecars are ready
+	// TODO: Implement waiting/registration
+	if *requestedHooks != 0 {
+		panic("Waiting mechanism for hookSidecars is not implemented")
```

Comment:
perfect, this is exactly where we should be pausing for the sidecar connections :+1:

Reply:
:+1: